src/store/Utils/index.ts

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+
+
+export const fetchFromApi = createAsyncThunk("fetchFromApi", async (url: string) =>{
+    const res = await axios.get(url)
+    if (res.status) {
+       return res.data
+    }
+    return console.log('need to return "NO DATA" page. ')
```

Comment:
wrap with try and catch like that:

try{
   const res = await axios.get(url)
    if (res.status) {
       return res.data
    }
} catch (err) {
      errorHandler();
    }